src/application/service/reports/GetReports.ts

```diff
@@ -0,0 +1,11 @@
+import { TestTypeId } from '../../../domain/model/test/testType/TestTypeId';
+import { ReportRepository } from '../../../domain/model/reports/ReportRepository';
+import { ReportTestResult } from '../../../domain/model/reports/ReportTestResult';
+
+export class GetReports {
+  constructor(private reportRepository: ReportRepository) {}
+
+  async getTestResults(testTypeId: TestTypeId): Promise<Array<ReportTestResult>> {
```

Comment:
Neatpicking: I would call this byTestTypeId to follow the same convention as other `Get*` classes